models/shipping-method/src/zone-rate/transformers.ts

```diff
@@ -0,0 +1,19 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TZoneRate, TZoneRateGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TZoneRate, TZoneRate>('default', {
+    buildFields: ['zone', 'shippingRates'],
+  }),
+  rest: Transformer<TZoneRate, TZoneRate>('rest', {
+    buildFields: ['zone', 'shippingRates'],
+  }),
+  graphql: Transformer<TZoneRate, TZoneRateGraphql>('graphql', {
+    buildFields: ['zone', 'shippingRates'],
```

Comment:
Similar to the previous comment, `ZoneRate` has `zoneRef`, and the `zone` field is a whole `Zone`. We might consider allowing the consumer to take a whole `Zone` in and then create the reference type as needed for both `rest` and `graphql` transformations.

![image](https://github.com/commercetools/test-data/assets/15024562/b0640ae4-aafa-41a2-b881-8dc7e887fc02)